### PR TITLE
MODSOURMAN-724 SRM does not process and save error records

### PR DESCRIPTION
**Pull request by Abdulkhakimov**
## Purpose
Link should be clickable when we click, but when SRS send DI error event records without parsed record have not information about title.

## Approach

- Added default title when we receive title without any data

## Learning
[MODSOURMAN-724](https://issues.folio.org/browse/MODSOURMAN-724)

### CODE REVIEW

File: mod-source-record-manager-server/src/main/java/org/folio/verticle/consumers/util/MarcImportEventsHandler.java [hunk 1]
```diff
@@ -104,6 +106,8 @@ public void handle(JournalService journalService, DataImportEventPayload eventPa
   private Future<JournalRecord> populateRecordTitleIfNeeded(JournalRecord journalRecord,
                                                             DataImportEventPayload eventPayload) {
     var entityType = journalRecord.getEntityType();
+    Optional.ofNullable(journalRecord.getTitle())
+      .ifPresentOrElse(title -> {}, () -> journalRecord.setTitle(NO_MARC_TITLE_MESSAGE));
```
Comment (SerhiiNosko) [inline mod-source-record-manager-server/src/main/java/org/folio/verticle/consumers/util/MarcImportEventsHandler.java:110]: I think we can simplify this, as option StringUtils and java trinary operator

Comment (SerhiiNosko) [inline mod-source-record-manager-server/src/main/java/org/folio/verticle/consumers/util/MarcImportEventsHandler.java:110]: In the method we use titleExtractor to populate journalRecord with title, so I think better to add this check at the end of method

Comment (Abdulkhakimov) [reply mod-source-record-manager-server/src/main/java/org/folio/verticle/consumers/util/MarcImportEventsHandler.java]: Done

Comment (Abdulkhakimov) [reply mod-source-record-manager-server/src/main/java/org/folio/verticle/consumers/util/MarcImportEventsHandler.java]: Done thank you

**Review by SerhiiNosko — APPROVED** [review]

**Review by AivarIUsupov — APPROVED** [review]

**Review by RuslanLavrov — APPROVED** [review]

**Pull request merged**